config/testgrids/kubernetes/presubmits/config.yaml

```diff
@@ -147,6 +147,8 @@ dashboards:
     base_options: width=10
   - name: pull-kubernetes-node-e2e
     test_group_name: pull-kubernetes-node-e2e
+  - name: pull-kubernetes-node-e2e-kubetest2
```

Comment:
I'm not sure if these are used, but looking at the other entries seems like this should also work.

In some other jobs I've seen annotations directly on the job e.g. https://github.com/kubernetes/test-infra/blob/3c87dfedd57e2dcad764a0d7fbb4343c7ca02a24/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L103-L147

```
annotations:
      testgrid-dashboards: presubmits-kubernetes-nonblocking
      testgrid-tab-name: pull-kubernetes-node-e2e-kubetest2
```

Reply:
Yes, I think both should work. One advantage what I can see of adding in `config.yaml` is that we can also configure the width of test status tab.